modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -996,6 +1007,26 @@ private void _validateName(
 		}
 	}
 
+	private void _validateRelationship(long objectFieldId, boolean required)
+		throws ObjectFieldRelationshipTypeException {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipPersistence.fetchByObjectFieldId2(objectFieldId);
+
+		if ((objectRelationship != null) &&
+			StringUtil.equals(
+				objectRelationship.getDeletionType(),
+				ObjectRelationshipConstants.DELETION_TYPE_DISASSOCIATE) &&
+			StringUtil.equals(
+				objectRelationship.getType(),
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY) &&
```

Comment:
This validation doesn't depend on relationship type.

Reply:
The feature is only to relationship type oneToMany, right? I validation the type oneToMany because of this.